src/app/js/formats/distributionChart/global-radar-chart/RadarChartAdmin.js

```diff
@@ -75,10 +76,86 @@ class RadarChartAdmin extends Component {
         updateAdminArgs('scale', scale, this.props);
     };
 
+    setColors = (_, colors) => {
+        updateAdminArgs('colors', colors, this.props);
+    };
+
+    setColorsWithColorPicker = () => {
+        updateAdminArgs('colors', this.getColorsFromPicker(), this.props);
+    };
+
+    getColorsFromPicker() {
+        let numberOfPickers = this.getColorsArray().length;
+        let res = '';
+        for (var i = 0; i < numberOfPickers; i++) {
+            if (i == 0) {
+                res += this.state.colors[i].color;
+            } else {
+                res += ' ' + this.state.colors[i].color;
+            }
+        }
+        this.setColors;
```

Comment:
This line does nothing.

Maybe
```suggestion
        this.setColors();
```
?

Reply:
Oh yes it does nothing, this one should be deleted